Documentation/security/security.rst

```diff
@@ -4,10 +4,10 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
-.. _security_tutorial_root:
+.. _security_root:
 
-Network Policy Security Tutorials
----------------------------------
+Securing networks with Cilium
```

Comment:
Nit: Other entries at that level of the ToC use title case. I also [recommend title case](https://docs.cilium.io/en/latest/contributing/docs/docsstyle/#titles) in general, to one day achieve better consistency, although we're far from there. Anyway. Not a strong preference, this one is mostly FYI.